src/utils/utils.js

```diff
@@ -46,3 +44,18 @@ export const addEmoji = (votes) => {
     return `👎${formatVotes}`;
   }
 };
+
+export const addAdjective = (articleVotes) => {
+  if (articleVotes > 0 && articleVotes !== 1) {
+    return "upvotes - what a great article. More like this below.";
+  } else if (articleVotes <= -2) {
+    return "downvotes - oh no!";
+  }
+  else if (articleVotes === 0) {
+    return "upvotes - it divides opinion!";
+  } else if (articleVotes === 1) {
+    return "upvote - must be worth reading";
+  } else {
+    return "downvote - oh no!";
+  }
+};
```

Comment:
Nice, but as this function is more specific to votes, I would recommend adding that into its name to be clearer on what this function actually does. Maybe something like addVoteAdjective?